mm2src/mm2_main/src/lp_ordermatch.rs

```diff
@@ -3561,6 +3564,13 @@ async fn process_maker_reserved(ctx: MmArc, from_pubkey: H256Json, reserved_msg:
                     connected: None,
                     last_updated: now_ms(),
                 };
+
+                ctx.event_stream_manager
+                    .send_fn(OrderStatusStreamer::derive_streamer_id(), || {
+                        OrderStatusEvent::TakerMatch(taker_match.clone())
+                    })
+                    .ok();
```

Comment:
I see that we always use `.ok()` with `send` and `send_fn`, shouldn't this error be logged somewhere instead, you can use `error_log_passthrough` or a similar approach if `send` and `send_fn` have to return an error.

Reply:
Agree to not log anything then, thanks